packages/cli/src/commands/labs/index.ts

```diff
@@ -0,0 +1,87 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+import {sortBy} from 'lodash'
+
+interface Features {
+  currentUser: any,
+  user: any,
+  app?: any,
+}
+
+function printJSON(features: Heroku.Account | Heroku.AccountFeature | Heroku.AppFeature) {
+  ux.log(JSON.stringify(features, null, 2))
+}
+
+function printFeatures(features: any) {
+  features = sortBy(features, 'name')
+  const longest = Math.max.apply(null, features.map((f: Record<string, string>) => f.name.length))
+  for (const f of features) {
+    let line = `${f.enabled ? '[+]' : '[ ]'} ${f.name!.padEnd(longest)}`
+    if (f.enabled) line = color.green(line)
+    line = `${line}  ${f.description}`
+    ux.log(line)
+  }
+}
+
+export default class LabsIndex extends Command {
+  static description = 'list experimental features'
+  static topic = 'labs'
+
+  static args = {
+    app: Args.string({required: false}),
+  }
+
+  static flags = {
+    app: flags.app({required: false}),
+    json: flags.boolean({description: 'display as json', required: false}),
+  }
+
+  async run() {
+    const {args, flags} = await this.parse(LabsIndex)
+    const [currentUserResponse, userResponse, appResponse] = await Promise.all([
+      this.heroku.get<Heroku.Account>('/account'),
+      this.heroku.get<Heroku.AccountFeature>('/account/features'),
+      (args.app || flags.app) ? this.heroku.get<Heroku.AppFeature>(`/apps/${args.app ? args.app : flags.app}/features`) : null,
+    ])
+
+    let app = null
+    const currentUser = currentUserResponse.body
+    const user = userResponse.body
+
+    const features: Features = {
+      currentUser,
+      user,
+    }
+
+    // makes sure app isn't added to json object if null
+    // eslint-disable-next-line no-negated-condition
+    if (appResponse !== null) {
```

Comment:
An alternate syntax that shortens the expression and accounts for `undefined` can be `const app = appResponse?.body`